frontend/components/bc/BcMaintenanceBanner.vue

```diff
@@ -0,0 +1,47 @@
+<script setup lang="ts">
+import { warn } from 'vue'
+
+const { public: { maintenanceTS } } = useRuntimeConfig()
+const { tick } = useInterval(60)
+const { t: $t } = useI18n()
+
+const maintenanceLabel = computed(() => {
+  if (!maintenanceTS) {
+    return
+  }
+  const parsed = typeof maintenanceTS === 'number' ? maintenanceTS : parseInt(maintenanceTS)
+  if (isNaN(parsed)) {
+    warn('NUXT_PUBLIC_MAINTENANCE_TS is not convertable to an intager, a unix ts is expected')
+    return undefined
+  } else if (parsed === 0) {
+    return
+  }
+  const ts = new Date(parseInt(maintenanceTS) * 1000).getTime()
```

Comment:
Would `parsed` fit here, instead of `parseInt(maintenanceTS)` ?

Reply:
yes that was the plan, but somehow got lost ;-)